### DISCUSSION TITLE
How to build libopenvino_intel_npu_plugin.so?

### DISCUSSION THREAD

**Discussion by kminemur** [openvinotoolkit/openvino#23060] [Q&A]
I followed default build steps, but not able to generate libopenvino_intel_npu_plugin.so.

# build steps
git clone https://github.com/openvinotoolkit/openvino.git
cd openvino/
git submodule update --init --recursive
chmod +x install_build_dependencies.sh
sudo ./install_build_dependencies.sh
mkdir build && cd build
cmake -DENABLE_PROFILING_ITT=ON -DCMAKE_BUILD_TYPE=Release -DENABLE_INTEL_NPU=ON ..
make --jobs=$(nproc --all)

**Comment by ilya-lavrenov** [comment]
Source code for NPU plugin is private.
It will be published soon (hope, before 2024.1 release)

CC @PatrikStepan

**Comment by kminemur** [comment]
Thanks for the reply
